HACKING.md

```diff
@@ -49,46 +51,89 @@ The parser transforms Giza-style YAML files using the `gizaparser`
 package. This uses the `flutter` library to deserialize the YAML files
 into Python classes, and check types to ensure there are no errors.
 
-### `parser.py`
-
+#### `parser.py`
 
-
-### `rstparser.py`
+#### `rstparser.py`
 
 `docutils`-interfacing components of the parser.
 
-### `gizaparser/`
+#### `gizaparser/`
 
 Each module in this package contains the infrastructure to parse a category
 of Giza YAML file. The `gizaparser.nodes` module contains generally-applicable
 helper classes.
 
-## Types & Tools
-### `flutter.py`
-### `types.py`
-### `util.py`
+### Types & Tools
+
+#### `flutter.py`
+
+#### `types.py`
+
+#### `util.py`
+
+## Developing Snooty
+
+Run the following to install the necessary tools:
+
+```shell
+python3 -m pip install flit virtualenv
+```
 
-# Developing Snooty
 Use [Flit](https://flit.readthedocs.io/en/latest/) to install Snooty. The module will be symlinked (via `-s`) to allow for testing changes without reinstalling the module.
+
 ```shell
 flit install -s
 ```
 
-## Running tests
+### Running tests
+
 To run tests for a specific file:
+
 ```shell
 . .venv/bin/activate
 pytest snooty/test_<file>.py
 ```
 
 ### Code Coverage
+
 Install [Coverage](https://coverage.readthedocs.io/en/v4.5.x/). After running tests via `make format test`, run:
+
 ```shell
 coverage html
 ```
+
 This will generate an HTML representation of code coverage throughout the repo that can be viewed in the browser.
 
-# Problem Areas
+### Release Process
+
+To release snooty, do the following:
+
+* Ensure that the "Unreleased" section of CHANGELOG.md is up-to-date.
+
+* Run `make cut-release BUMP_TO_VERSION="<new_version>"`, and make note of the release name that
```

Comment:
Are there any standards we want to follow for version naming? I don't have much experience with it, so perhaps it could be useful to either explain our approach or link out to another resource somewhere in this document. I think Sue's comment addresses this as well!

Reply:
Added more details!